artifactory.py

```diff
@@ -2463,7 +2463,7 @@ def _get_all(self, lazy: bool, url=None, key="name", cls=None):
         "return: A list of found objects
         """
         if cls is Project:
-            request_url = self.drive.rstrip("/artifactory") + url
+            request_url = self.drive.replace("/artifactory", "") + url
```

Comment:
May be something like rexegp? The `replace` doesn't care if it meats the string at the middle and we care only about the tail here
```suggestion
            request_url = re.sub(r"/artifactory$", "", self.drive) + url
```